2025/05/06 15:07:13 [error] 35132#37600: *4 CreateFile() "E:\nginx\nginx-1.28.0/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/06/09 11:05:55 [error] 39204#12280: *4 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/06/09 11:09:27 [emerg] 22136#31256: host not found in upstream "webservers" in D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/conf/nginx.conf:65
2025/06/09 11:09:45 [emerg] 25104#25128: host not found in upstream "webservers" in D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/conf/nginx.conf:65
2025/06/09 11:12:13 [emerg] 27388#7988: host not found in upstream "webservers" in D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/conf/nginx.conf:65
2025/06/09 11:23:43 [emerg] 39320#19612: host not found in upstream "webservers" in D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/conf/nginx.conf:65
2025/06/09 11:24:38 [emerg] 39760#29784: host not found in upstream "webservers" in D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/conf/nginx.conf:65
2025/06/09 11:25:32 [error] 39204#12280: *32 "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/api/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /api/ HTTP/1.1", host: "localhost"
2025/06/09 11:27:36 [emerg] 7396#24732: host not found in upstream "webservers" in D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/conf/nginx.conf:65
2025/06/09 11:34:13 [emerg] 6128#9940: CreateDirectory() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/temp/client_body_temp" failed (3: The system cannot find the path specified)
2025/06/09 11:36:55 [notice] 8020#29728: signal process started
2025/06/09 11:38:45 [notice] 17268#39836: signal process started
2025/06/09 11:43:48 [error] 27312#37380: *13 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/ HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/", host: "localhost"
2025/06/09 11:46:23 [error] 27312#37380: *20 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/09 11:46:26 [error] 27312#37380: *23 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/09 11:46:34 [error] 27312#37380: *23 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /user/ HTTP/1.1", upstream: "http://127.0.0.1:8080/user/", host: "localhost"
2025/06/09 15:36:48 [error] 27312#37380: *35 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 16:01:18 [error] 27312#37380: *53 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 16:12:40 [error] 27312#37380: *62 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 16:15:42 [error] 27312#37380: *72 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 16:15:50 [error] 27312#37380: *76 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/09 16:15:58 [error] 27312#37380: *81 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/09 16:27:28 [error] 27312#37380: *83 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 16:35:49 [error] 27312#37380: *102 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 16:48:39 [error] 27312#37380: *113 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 17:18:22 [error] 27312#37380: *173 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 17:26:49 [error] 27312#37380: *230 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 17:35:12 [error] 27312#37380: *269 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 17:45:58 [error] 27312#37380: *278 access forbidden by rule, client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 17:48:09 [error] 33032#5268: *2 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 17:48:09 [error] 33032#5268: *1 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/09 17:48:19 [error] 33032#5268: *17 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/10 11:39:11 [error] 33032#5268: *126 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/10 11:43:25 [error] 15320#31840: *1 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/10 11:53:58 [error] 15320#31840: *42 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/10 11:54:07 [error] 15320#31840: *48 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/10 11:54:50 [error] 15320#31840: *90 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/10 15:16:10 [notice] 19940#28756: signal process started
2025/06/10 15:17:06 [error] 24352#26796: *196 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/10 15:19:31 [notice] 34528#4324: signal process started
2025/06/10 15:26:12 [notice] 32152#5136: signal process started
2025/06/10 15:27:17 [notice] 40008#12756: signal process started
2025/06/10 15:27:57 [error] 24352#26796: *200 CreateFile() "D:\CodeProjects\IdeaProjects\sky-take-out\nginx-1.28.0/html/sky/.well-known/appspecific/com.chrome.devtools.json" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", host: "localhost"
2025/06/14 17:38:03 [alert] 15100#9980: worker process 19600 exited with code C0000142
2025/06/14 17:38:03 [alert] 15100#9980: could not respawn worker
2025/06/24 17:50:09 [error] 23540#17136: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/24 17:50:11 [error] 23540#17136: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/24 17:56:59 [error] 23540#17136: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/24 17:57:01 [error] 23540#17136: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/06/24 18:00:53 [error] 23540#17136: *21 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while proxying upgraded connection, client: 127.0.0.1, server: localhost, request: "GET /ws/h0fgmvqhgmq HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/h0fgmvqhgmq", host: "localhost"
2025/06/24 18:16:05 [error] 23540#17136: *101 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while proxying upgraded connection, client: 127.0.0.1, server: localhost, request: "GET /ws/t04tanllhk HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/t04tanllhk", host: "localhost"
2025/06/24 18:26:06 [error] 23540#17136: *115 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while proxying upgraded connection, client: 127.0.0.1, server: localhost, request: "GET /ws/n4cxynplw HTTP/1.1", upstream: "http://[::1]:8080/ws/n4cxynplw", host: "localhost"
2025/06/25 11:14:36 [error] 23540#17136: *133 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/kv2qys09qpe HTTP/1.1", upstream: "http://[::1]:8080/ws/kv2qys09qpe", host: "localhost"
2025/06/25 11:24:05 [error] 23540#17136: *224 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while proxying upgraded connection, client: 127.0.0.1, server: localhost, request: "GET /ws/rggl3791xqh HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/rggl3791xqh", host: "localhost"
2025/06/25 11:32:03 [error] 23540#17136: *252 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while proxying upgraded connection, client: 127.0.0.1, server: localhost, request: "GET /ws/cx0toi9kk3p HTTP/1.1", upstream: "http://[::1]:8080/ws/cx0toi9kk3p", host: "localhost"
2025/06/25 11:34:16 [error] 23540#17136: *258 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://127.0.0.1:8080/api/shop/status", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *260 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/turnoverStatistics?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://[::1]:8080/api/report/turnoverStatistics?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *260 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/turnoverStatistics?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://localhost/api/report/turnoverStatistics?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *261 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/userStatistics?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://127.0.0.1:8080/api/report/userStatistics?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *261 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/userStatistics?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://localhost/api/report/userStatistics?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *259 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/ordersStatistics?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://[::1]:8080/api/report/ordersStatistics?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *259 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/ordersStatistics?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://localhost/api/report/ordersStatistics?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *256 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/top10?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://127.0.0.1:8080/api/report/top10?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *256 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/report/top10?begin=2025-06-18&end=2025-06-24 HTTP/1.1", upstream: "http://localhost/api/report/top10?begin=2025-06-18&end=2025-06-24", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:16 [error] 23540#17136: *267 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/jo7rawugug HTTP/1.1", upstream: "http://[::1]:8080/ws/jo7rawugug", host: "localhost"
2025/06/25 11:34:16 [error] 23540#17136: *267 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/jo7rawugug HTTP/1.1", upstream: "http://localhost/ws/jo7rawugug", host: "localhost"
2025/06/25 11:34:18 [error] 23540#17136: *258 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://[::1]:8080/api/shop/status", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:44 [error] 23540#17136: *278 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/ogrqx7sjh0k HTTP/1.1", upstream: "http://localhost/ws/ogrqx7sjh0k", host: "localhost"
2025/06/25 11:34:44 [error] 23540#17136: *279 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/businessData HTTP/1.1", upstream: "http://localhost/api/workspace/businessData", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:44 [error] 23540#17136: *280 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/overviewOrders HTTP/1.1", upstream: "http://localhost/api/workspace/overviewOrders", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:44 [error] 23540#17136: *281 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/overviewDishes HTTP/1.1", upstream: "http://localhost/api/workspace/overviewDishes", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:44 [error] 23540#17136: *282 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/overviewSetmeals HTTP/1.1", upstream: "http://localhost/api/workspace/overviewSetmeals", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:46 [error] 23540#17136: *274 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/order/conditionSearch?page=1&pageSize=10&status=2 HTTP/1.1", upstream: "http://127.0.0.1:8080/api/order/conditionSearch?page=1&pageSize=10&status=2", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:46 [error] 23540#17136: *274 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/order/conditionSearch?page=1&pageSize=10&status=2 HTTP/1.1", upstream: "http://localhost/api/order/conditionSearch?page=1&pageSize=10&status=2", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:46 [error] 23540#17136: *275 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://[::1]:8080/api/shop/status", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:34:46 [error] 23540#17136: *275 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://localhost/api/shop/status", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:35:47 [error] 23540#17136: *289 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://localhost/api/shop/status", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:35:47 [error] 23540#17136: *290 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/businessData HTTP/1.1", upstream: "http://localhost/api/workspace/businessData", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:35:47 [error] 23540#17136: *291 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/overviewOrders HTTP/1.1", upstream: "http://localhost/api/workspace/overviewOrders", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:35:47 [error] 23540#17136: *292 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/overviewDishes HTTP/1.1", upstream: "http://localhost/api/workspace/overviewDishes", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:35:47 [error] 23540#17136: *293 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/workspace/overviewSetmeals HTTP/1.1", upstream: "http://localhost/api/workspace/overviewSetmeals", host: "localhost", referrer: "http://localhost/"
2025/06/25 11:37:24 [error] 23540#17136: *332 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while proxying upgraded connection, client: 127.0.0.1, server: localhost, request: "GET /ws/6d4dwmqaczo HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/6d4dwmqaczo", host: "localhost"
2025/06/25 11:52:42 [notice] 20732#9456: signal process started
2025/06/25 12:00:13 [notice] 8156#22848: signal process started
2025/06/25 12:04:39 [notice] 16968#9700: signal process started
2025/06/25 12:08:25 [error] 9780#20952: *26 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while proxying upgraded connection, client: 127.0.0.1, server: localhost, request: "GET /ws/uinoia6sbe HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/uinoia6sbe", host: "localhost"
